diff --git a/myosuite/envs/myo/myobase/__init__.py b/myosuite/envs/myo/myobase/__init__.py
index 64198fa..5522174 100644
--- a/myosuite/envs/myo/myobase/__init__.py
+++ b/myosuite/envs/myo/myobase/__init__.py
@@ -502,10 +502,10 @@ register_env_with_variants(id='myoTorsoExoFixed-v0',
             'target_jnt_range': {'LB_wrapjnt_t1':(0, 0),'LB_wrapjnt_t2':(0, 0),'LB_wrapjnt_r3':(0, 0),
                                  'flex_extension':(-1.39626, -1.39626),'lat_bending':(-0, 0),'axial_rotation':(0, 0), #was -0.1, 0.1
                                  'Abs_t1':(0, 0),'Abs_t2':(0,0),'Abs_r3':(0, 0),
-                                 'L4_L5_FE':(-1, 1),'L4_L5_LB':(0, 0),'L4_L5_AR':(0, 0),
-                                 'L3_L4_FE':(-1, 1),'L3_L4_LB':(0, 0),'L3_L4_AR':(0, 0),
-                                 'L2_L3_FE':(-1, 1),'L2_L3_LB':(0, 0),'L2_L3_AR':(0, 0),
-                                 'L1_L2_FE':(-1, 1),'L1_L2_LB':(0, 0),'L1_L2_AR':(0, 0),},
+                                 'L4_L5_FE':(0, 0),'L4_L5_LB':(0, 0),'L4_L5_AR':(0, 0), #all FE were -1 to 1
+                                 'L3_L4_FE':(0, 0),'L3_L4_LB':(0, 0),'L3_L4_AR':(0, 0),
+                                 'L2_L3_FE':(0, 0),'L2_L3_LB':(0, 0),'L2_L3_AR':(0, 0),
+                                 'L1_L2_FE':(0, 0),'L1_L2_LB':(0, 0),'L1_L2_AR':(0, 0),},
             'normalize_act': True,
             'frame_skip': 5,
         }
